doc/conf_ref.rst

```diff
@@ -263,6 +271,22 @@ configuration.
 
     When enabled, remote SSL connections are verified. If the client can not be authenticated connecting fails and the given repo is not used further. On False, SSL connections can be used but are not verified. Default is True.
 
+.. _sslclientcert-label:
+
+``sslclientcert``
+    string
+
+    Path to the SSL client certificate used to connect to remote sites.
+    Defaults to none.
+
+.. _sslclientkey-label:
+
+``sslclientkey``
+    string
+
+    Path to the SSL client key used to connect to remote sites.
+    Defaults to none.
```

Comment:
in conf_ref so far we used to write "Empty by default." (For example in username/password you have added to API)